### DISCUSSION TITLE
Urgent [Bug] Comment being shown on UI

### DISCUSSION THREAD

**Discussion by GoelDarpan** [dotnet/docfx#9288] [Q&A]
**Describe the bug**
[//]: # "APIEndpoint:Start" in my index.md file started showing on my UI, It didn't come earlier did anything change?
I need help in resolving this and need to know why this happened.

In Index.html - 
![image](https://github.com/dotnet/docfx/assets/147417938/c8673c15-4330-454a-b07e-1c31b60c589b)

On UI - 
![image](https://github.com/dotnet/docfx/assets/147417938/eb73f7be-97c3-4c0f-8f46-acb3be1b957e)

**Expected behavior**

**Context (please complete the following information):**
- OS: Windows

**Additional context**

**Comment by yufeih** [comment]
Does `index.md` file render correctly on GitHub or other markdown editors?

**Reply by KalleOlaviNiemitalo** [reply]
I don't know how Markdown editors handle the line

```markdown
[//]: # "APIEndpoint:Start"
```

but it at least seems to match the syntax for [link reference definition](https://spec.commonmark.org/0.30/#link-reference-definitions) in CommonMark 0.30:

- `[//]` is a [link label](https://spec.commonmark.org/0.30/#link-label)
- `#` is a [link destination](https://spec.commonmark.org/0.30/#link-destination) (the alternative that does not start with `<`)
- `"APIEndpoint:Start"` is a [link title](https://spec.commonmark.org/0.30/#link-title)

But… there was a bug <https://github.com/dotnet/docfx/issues/7850>.  @GoelDarpan, are you using a DocFX version in which that has been fixed?
